docs/NewProject.md

```diff
@@ -0,0 +1,44 @@
+﻿<!-- Copyright (c) Microsoft Corporation. All rights reserved.
+     Licensed under the MIT License. -->
+     
+     ## Adding a new project
+Do the following when adding a new project:
+
+### For all projects
+1. Add a project to the Axe.Windows Solution (`src\axe.windows.sln`).
+2. Right-click on the project and select Properties.
+2. In the Application tab, configure "Target Framework" to use the same .NET Framework version used by the `axe.windows.core` project.
+   - Currently .NET Framework 4.7.1 is the target. 
+3. In the build tab, set the following for both Debug and Release configurations:
+   1. "Warning level" to 4.
+   2. "Treat warnings as errors" to "All".
+
+### For *non-test* projects only
+1. Add the following NuGet packages in Visual Studio to enable signing and code analysis:
+   ```
+   Microsoft.VisualStudioEng.MicroBuild.Core
+   Microsoft.CodeAnalysis.FxCopAnalyzers
+   ```
+2. Close the solution and use your text editor to make the following changes to your `.csproj` file to properly configure the version and signing options:
+   1. Add the following line with the other `.cs` files):<br>
+   `<Compile Include="$(TEMP)\A11yInsightsVersionInfo.cs" />`
```

Comment:
unsure whether we need to change the text "A11yInsightsVersionInfo" or change the file name and the text.

Reply:
I'll open a PR to change the file to AxeWindowsVersionInfo.cs

---
In reply to: [299139845](https://github.com/microsoft/axe-windows/pull/88#discussion_r299139845) [](ancestors = 299139845)

Reply:
@RobGallo, PR is at [https://github.com/microsoft/axe-windows/pull/89](https://github.com/microsoft/axe-windows/pull/89)

---
In reply to: [299173222](https://github.com/microsoft/axe-windows/pull/88#discussion_r299173222) [](ancestors = 299173222,299139845)